.drone.jsonnet

```diff
@@ -724,7 +724,7 @@ local Pipeline(branch, platform, event, arch='amd64', server='10.6-enterprise')
       'docker exec -t mcs1 mariadb -e "create database if not exists test;"',
       // delay for manual debugging on live instance
       'sleep $${COMPOSE_DELAY_SECONDS:-1s}',
-      'docker exec -t mcs1 bash -c "cd ' + mtr_path + ' && ./mtr --extern socket=' + socket_path + ' --force --print-core=detailed --print-method=gdb --max-test-fail=0 --suite=columnstore/basic,columnstore/bugfixes"',
+      'docker exec -t mcs1 bash -c "cd ' + mtr_path + ' && ./mtr --skip-test=*mcol-5505-cpimport-parquet* --skip-test=fdb_api --extern socket=' + socket_path + ' --force --print-core=detailed --print-method=gdb --max-test-fail=0 --suite=columnstore/basic,columnstore/bugfixes"',
```

Comment:
@mariadb-AlanMologorsky We discussed you will disable those two only for rocky-8 b/c it fails running the tests with 3 nodes cluster.

Reply:
This skip arguments should are applied to   multi_node_mtr step, that is exactly rocky-8 multinode